pkg/client/rules.go

```diff
@@ -76,11 +104,18 @@ func (r *CortexClient) GetRuleGroup(ctx context.Context, namespace, groupName st
 
 // ListRules retrieves a rule group
 func (r *CortexClient) ListRules(ctx context.Context, namespace string) (map[string][]rwrulefmt.RuleGroup, error) {
-	path := "/api/prom/rules"
+	path := rulerAPIPath
+	if r.legacy {
+		path = legacyAPIPath
+	}
 	if namespace != "" {
 		path = path + "/" + namespace
 	}
 
+	log.WithFields(log.Fields{
+		"url": path,
+	}).Debugln("path built to request rule group")
```

Comment:
**nitpick:** I think the `doRequest` function logs the path of the request. If so, this log line should be removed.

Reply:
I agree. I'll remove it.